build.gradle

```diff
@@ -83,6 +86,8 @@ dependencies {
     }
     testCompile 'commons-io:commons-io:2.5'
 
+    // This kotlin version needs to be compatible with the 
+    // android plugin being used in the test runtime.
     testProjectRuntime "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.0"
```

Comment:
Bumping this dependency version is what was causing the failures in the ci. The test runtime kotlin version needs to be compatible with the older version of the android plugin. I didnt need to update it in the first place so its fine being set to ```1.2.0```